Alerting: Allow custom prefixes for OpsGenie notifications (#45016)
grafana/grafana: docs/sources/alerting/old-alerting/notifications.md

| API Key                   | The API Key as provided by Opsgenie for your configured Grafana integration.                                                                                                                                                               |
| Override priority         | Configures the alert priority using the `og_priority` tag. The `og_priority` tag must have one of the following values: `P1`, `P2`, `P3`, `P4`, or `P5`. Default is `False`.                                                               |
| Send notification tags as | Specify how you would like [Notification Tags]({{< relref "create-alerts.md/#notifications" >}}) delivered to Opsgenie. They can be delivered as `Tags`, `Extra Properties` or both. Default is Tags. See note below for more information. |

> | Override alias prefix     | In case multiple instances of grafana are connected to a single OpsGenie account, the alias prefix is a unique identifier for each instance (a-zA-Z0-9).                                                                                   |

Reviewer: ```suggestion
| Override alias prefix     | If multiple instances of Grafana are connected to a single OpsGenie account, the alias prefix is a unique identifier for each instance (a-zA-Z0-9).                                                                                   |
```
